ocaml-lsp-server/src/compl.ml

```diff
@@ -128,5 +121,56 @@ let complete doc position ~markdown =
       let range = range_prefix lsp_position prefix in
       List.map ~f:(fun entry -> `Replace (range, entry)) entries
   in
-  let items = List.mapi ~f:(item ~markdown) items in
+  let textDocument =
+    TextDocumentIdentifier.create ~uri:(Document.uri doc |> Lsp.Uri.to_string)
+  in
+  let compl_info =
+    CompletionParams.(
```

Comment:
Local open should be avoided.

Reply:
I thought your comment on local opens to my previous PR was only for `let open ... in`. Here I thought the readability doesn't suffer. Fixed.